extension/linkify-urls-in-code.js

```diff
@@ -28,7 +28,7 @@ window.linkifyURLsInCode = (() => {
 		.toArray()
 		.forEach(blob => {
 			if (hasURL(blob.innerHTML)) {
```

Comment:
> True that, I guess the initial author wanted to avoid running match() in general in case there is no link present.

But this is actually more expensive. Now you run two check when it's successful, instead of one. And when it doesn't exist, it's the same performance. Let's just remove that if statement.